ovnnbimp_test.go

```diff
@@ -475,4 +475,73 @@ func TestLoadBalancer(t *testing.T) {
 		t.Fatalf("error: lb deletion not done, total:%v", len(lb))
 	}
 	t.Logf("LB deletion done")
+	// test adding lb to lr and lswitch
```

Comment:
i'm prefer to split this big file to smaller pieces and add tests for specific files, like logical_switch_test.go load_balancer_test.go and so.
I think that this is more idiomatic with other go projects.

Reply:
Yeah will do